ksqldb-engine/src/main/java/io/confluent/ksql/topic/TopicCreateInjector.java

```diff
@@ -51,6 +52,11 @@
  */
 public class TopicCreateInjector implements Injector {
 
+  private static final String CLEANUP_POLICY_PRESENT_EXCEPTION =
+      String.format(
+          "Invalid config variable in the WITH clause: %s",
```

Comment:
It'd be good to explain _why_ it's invalid in the error message

Reply:
changed to:
```
Invalid config variable in the WITH clause: CLEANUP_POLICY.
The CLEANUP_POLICY config is automatically inferred based on the type of source (STREAM or TABLE).
Users can't set the CLEANUP_POLICY config manually.
```